src/InstanceService.ts

```diff
@@ -495,7 +504,12 @@ class InstanceService {
 
       const filePayload = await instance.account.service.conversation.createFileData(file, metadataPayload.id);
       const sentFile = await instance.account.service.conversation.send(conversationId, filePayload);
-      delete (sentFile.content as FileContent).data;
+
+      delete (sentFile.content as FileAssetContent).asset.cipherText;
```

Comment:
I am seeing those 3 lines of code many times. Might be a sign that we could extract that functionality in an helper function.
Something like `clearEncryptionData(asset)` or something similar (I'll let you find the right naming here)